src/libultra/io/contsetch.c

```diff
@@ -8,7 +8,7 @@ s32 osContSetCh(u8 ch) {
     } else {
         __osMaxControllers = ch;
     }
-    __osContLastCmd = -2;
+    __osContLastPoll = -2;
```

Comment:
This is a bit out of scope, but it looks like there are some macros in controller.h for this magic numbers.
We should make a clean up PR for stuff like this in the future